code/go/internal/validator/test/packages/missing_pipeline_dashes/manifest.yml

```diff
@@ -5,7 +5,8 @@ description: This package is good.
 version: 0.1.2
 release: beta
 conditions:
-  kibana.version: '^7.9.0'
+  kibana:
+    version: '^7.9.0'
```

Comment:
I assume the purpose of the new `expanded` test package is to test the case where the kibana version condition is specified like this:

```
kibana:
  version: ...
```

Given that, should we leave this line as-is (flattened syntax) to make sure both syntaxes are being tested?

Reply:
You're right. Fixed.